equinix/fabric_mapping_helper.go

```diff
@@ -252,6 +264,27 @@ func locationToFabric(locationList []interface{}) v4.SimplifiedLocation {
 	return sl
 }
 
+func virtualdeviceToFabric(virtualdeviceList []interface{}) v4.VirtualDevice {
+	vd := v4.VirtualDevice{}
+	for _, ll := range virtualdeviceList {
+		llMap := ll.(map[string]interface{})
+		tp := llMap["type"].(string)
+		ud := llMap["uuid"].(string)
+		vd = v4.VirtualDevice{Type_: tp, Uuid: ud}
+	}
+	return vd
+}
+
+func interfaceToFabric(interfaceList []interface{}) v4.ModelInterface {
```

Comment:
This isn't parsing all of what is in the schema for interface. You changed the type of id from String to Int but it's not being parsed here.

Reply:
I have parsed only parameters which are required for API call for the virtual device and interface model.

Reply:
We have the other values in the schema and in the published documentation. Anything we have exposed as usable needs to be included.